core/templates/filters/string-utility-filters/truncate-at-first-ellipsis.pipe.spec.ts

```diff
@@ -31,8 +31,12 @@ describe('Testing filters', function() {
 
   it('should truncate a string when it first sees a \'...\'', () => {
     expect(truncateAtFirstEllipsisPipe.transform('')).toEqual('');
-    expect(truncateAtFirstEllipsisPipe.transform(null)).toEqual('');
-    expect(truncateAtFirstEllipsisPipe.transform(undefined)).toEqual('');
+    expect(truncateAtFirstEllipsisPipe.transform(
+      // Use unknown type conversion to check input invalidity.
+      null as unknown as string)).toEqual('');
+    expect(truncateAtFirstEllipsisPipe.transform(
```

Comment:
I think we should remove these test cases if null and undefined values aren't expected in the transform function

Reply:
Done